### PR TITLE
Issue 2 - adding black panther

### PR DESCRIPTION
**Pull request by nerdstein**
References #2

### CODE REVIEW

**Comment by joelpheasant** [comment]
great job, but year is wrong.

File: index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
 <tr><th>Year</th><th>Movie</th></tr>
 <tr><td>2008</td><td>Iron Man</td></tr>
 <tr><td>2011</td><td>Captain America: The First Avenger</td></tr>
+<tr><td>2017</td><td>Black Panther</td></tr>
```
Comment (joelpheasant) [inline index.html]: Should be 2018

**Review by joelpheasant — CHANGES_REQUESTED** [review]
should be 2018

File: index.html (hunk 1 above)
Comment (nerdstein) [reply index.html]: Updated

**Review by joelpheasant — APPROVED** [review]
looks good

**Pull request merged**